src/flows/pipes/Notification/endpoints/Mailgun/index.ts

```diff
@@ -33,37 +37,57 @@ export default register => {
 	|> monitor["notify"](
     data: notification,
     endpoint: ((r) => {
-        apiKey = secrets.get(key: "${data.apiKey}")
-        http.post(
-            url: "https://api.mailgun.net/v3/${data.domain}/messages",
-            headers: {
-              "Content-type": "application/json",
-              "Authorization": "Basic api:\${apiKey}"
-            },
-            data: bytes(v: "{
-              \\"from\\": \\"Username mailgun@${data.domain}\\",
-              \\"to\\": \\"${data.email}\\",
-              \\"subject\\": \\"InfluxDB Alert\\",
-              \\"text\\": \\"\${ r._message }\\"
-            }"))
-    })
-  )`,
-    generateTestQuery: data => `
-    apiKey = secrets.get(key: "{data.apiKey}")
-    http.post(
-        url: "https://api.mailgun.net/v3/${data.domain}/messages",
+      apiKey = secrets.get(key: "${data.apiKey}")
+      auth = http.basicAuth(u: "api", p: "\${apiKey}")
+      url = "https://api.mailgun.net/v3/${data.domain}/messages"
+      data = strings.joinStr(arr: [
+          "from=${fromEmail}",
+          "to=${data.email}",
+          "subject=${subject}",
+          "text=\${r._message}"
```

Comment:
Seems like we want to pass `r._message` as a variable. If so, do we want to remove the `\` before the dollor sign? Maybe I'm wrong since I'm not familiar with the syntax here

Reply:
The r is a flux variable, being used for flux string templating.
Which happens to use the same syntax as javascript string templating, so the escaping is to keep it present for flux.